includes/db/Database.php

```diff
@@ -3606,6 +3606,18 @@ protected function indexNameCallback( $matches ) {
 		return $this->indexName( $matches[1] );
 	}
 
+	/**
+	 * Check to see if a named lock is available. This is non-blocking.
+	 *
+	 * @param string $lockName name of lock to poll
+	 * @param string $method name of method calling us
+	 * @return Boolean
+	 * @since 1.20
+	 */
+	public function lockIsFree( $lockName, $method ) {
+		return true;
+	}
+
```

Comment:
?

Reply:
This method was backported from 1.20 but only to `DatabaseMysqlBase` class. It is thus missing from the parent class and causes an error if something tries to call it in SQLite integration tests.